hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -132,15 +132,15 @@
           name: "ENABLE",
           mubi: true,
           desc: '''
-                Setting this field to 0xA will enable the ENTROPY_SRC module.
+                Setting this field to kMuBi4True will enable the ENTROPY_SRC module.
```

Comment:
I made a similar comment on another mubi PR, I think for the hjson/generated docs it would be better if this was the actual value (0xA) instead of kMuBi4True.  But in talking with @tjaychen he mentioned it's possible for the value to be other than 0xA so there is a concern about docs being stale/incorrect.  He and I reached the same conclusion that the ideal solution is for this to be a generated/preprocessed value, and he's making an issue for it.  TBD when we actually do this though, so using kMuBi4True for now is OK to me.

Reply:
Can EDN be also added into this PR, given its exactly the same topic?